### PR TITLE
[STAKAN-193] Balance block. APY. Staking/unstaking links

### PR DESCRIPTION
**Pull request by fox-on-rocket**
https://ankrnetwork.atlassian.net/browse/STAKAN-193

Merge after https://github.com/Ankr-network/stkr-web3/pull/365

### CODE REVIEW

File: src/modules/stake/components/StakeForm/StakeForm.tsx [hunk 1]
```diff
@@ -33,8 +33,8 @@ export interface IStakeFormComponentProps {
   maxAmount?: number;
   loading: boolean;
   currency?: string;
-  renderStats?: (amount: number) => ReactNode;
-  renderFooter?: (amount: number) => ReactNode;
+  renderStats?: (amount?: number | string) => ReactNode;
+  renderFooter?: (amount?: number | string) => ReactNode;
```
Comment (BigBadAlien) [inline src/modules/stake/components/StakeForm/StakeForm.tsx:37]: Let's unify the type to string. Number can come only from initialValues props of Form

Comment (fox-on-rocket) [reply src/modules/stake/components/StakeForm/StakeForm.tsx:37]: Actual fix: https://github.com/Ankr-network/stkr-web3/pull/365

File: src/modules/stake-polygon/actions/fetchAPY.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import BigNumber from 'bignumber.js';
+import { createAction as createSmartAction } from 'redux-smart-actions';
+import { PolygonSDK } from '../api/PolygonSDK';
+
+export const fetchAPY = createSmartAction<RequestAction<BigNumber, BigNumber>>(
+  'polygon/fetchAPY',
+  () => ({
+    request: {
+      promise: (async (): Promise<BigNumber> => {
+        const sdk: PolygonSDK = PolygonSDK.getInstance();
```
Comment (Antroll) [inline src/modules/stake-polygon/actions/fetchAPY.ts:11]: Why just not like that?
```suggestion
        const sdk = PolygonSDK.getInstance();
```

**Comment by Antroll** [comment]
@fox-on-rocket, please sync up this branch with a branch from #365 for review.

**Comment by fox-on-rocket** [comment]
@Antroll, done

**Review by Antroll — APPROVED** [review]

**Pull request merged**